externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -48,7 +48,14 @@ class ExecuteStatement(
   private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
   override def getOperationLog: Option[OperationLog] = Option(operationLog)
 
-  private val operationListener: SQLOperationListener = new SQLOperationListener(this, spark)
+  private val conf: KyuubiConf = getSession.sessionManager.getConf
+
+  private val operationListener: Option[SQLOperationListener] =
```

Comment:
we can make it settable via sql command

Reply:
@cxzl25 seems this comment has not been addressed ? can you take a look ?

Reply:
It should be solved, I add set true/false in UT to test.
Thanks for your review.